src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -448,4 +470,13 @@ sub-emptyroom/
             sub-emptyroom_ses-20170801_task-noise_meg.json
 ```
 
-`TaskName` in the `*_meg.json` file should be set to "noise".
+Example for cases resembling strategy 2:
+
+```Text
+sub-control01/
+sub-control02/
+    sub-control01_task-bart_meg.ds
```

Comment:
aren't you missing the modality `meg/` here?

Reply:
yes, thanks - see https://github.com/bids-standard/bids-specification/pull/480/files#r434364440